fixDeploy.mjs

```diff
@@ -0,0 +1,36 @@
+// bug handler copied from https://github.com/facebook/create-react-app/issues/12503
+
+import fse from 'fs-extra';
+import path from 'path';
+import { fileURLToPath, URL } from 'url';
+
+(async () => {
+	const repoRootUrl = new URL('./', import.meta.url);
+	const repoRoot = repoRootUrl.pathname;
+	const repoFolder = repoRoot.split('/').filter(Boolean).pop() ?? '/';
+	const sourceFolder = path.join(repoRoot, 'build/static/js');
+	// const destinationFolder = path.join(sourceFolder, 'static/js');
+
+	if (!fse.existsSync(sourceFolder)) {
+		throw new Error(
+			`Failed to handle webpack path bug. Source folder (${sourceFolder}) does not exist.`
+		);
+	}
+
+	// await fse.ensureDir(destinationFolder);
+	const staticJsFiles = await fse.readdir(sourceFolder);
+
+	// Move all .js files that do not start with "main." or "web-worker." as those two get correctly loaded directly from static/js.
```

Comment:
```suggestion
```

are we now really moving the files somewhere ?

Reply:
```suggestion
	// Fix paths in all .js files that do not start with "main." or "web-worker." as those two get correctly loaded directly from static/js.
```